api/src/main/java/com/velocitypowered/api/proxy/Player.java

```diff
@@ -205,4 +205,18 @@ default void sendMessage(net.kyori.text.Component component) {
    * @param hash the SHA-1 hash value for the resource pack
    */
   void sendResourcePack(String url, byte[] hash);
```

Comment:
Can convert this into a `default` method, calling the new one with `required` as `false`.

Reply:
Will do that when I roll 20w46a